### PR TITLE
Minimal rewrite of README to explain Flux v2

### PR DESCRIPTION
**Pull request by squaremo**
- Adjust the description so it talks about the whole system (Flux) as
   one thing
 - Describe what GitOps Toolkit is, and how it relates to Flux v2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,16 +49,28 @@ gotk check --pre
 
 ## Get started
 
-To get started with the GitOps Toolkit, start [browsing the documentation](https://toolkit.fluxcd.io)
-or get started with one of the following guides:
+To get started with the Flux, start [browsing the
```
Comment (dholbach) [inline README.md:52]:
```suggestion
To get started with Flux, start [browsing the
```

**Comment by squaremo** [comment]
(I'll commit suggested changes for now and rebase later)

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
-# GitOps Toolkit
+# Flux version 2
```
Comment (stefanprodan) [inline README.md:1]: I would call it Flux and update the fluxcd/flux readme and set there v1. See https://github.com/linkerd/linkerd2

File: README.md [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
 [![e2e](https://github.com/fluxcd/toolkit/workflows/e2e/badge.svg)](https://github.com/fluxcd/toolkit/actions)
 [![report](https://goreportcard.com/badge/github.com/fluxcd/toolkit)](https://goreportcard.com/report/github.com/fluxcd/toolkit)
 [![license](https://img.shields.io/github/license/fluxcd/toolkit.svg)](https://github.com/fluxcd/toolkit/blob/main/LICENSE)
 [![release](https://img.shields.io/github/release/fluxcd/toolkit/all.svg)](https://github.com/fluxcd/toolkit/releases)
 ![overview](docs/diagrams/gitops-toolkit.png)
 
-The GitOps Toolkit is a set of composable APIs and specialized tools
-that can be used to build a Continuous Delivery platform on top of Kubernetes.
+Flux is a tool for keeping Kubernetes clusters in sync with
+configuration in Git repositories, and automating updates to
```
Comment (stefanprodan) [inline README.md:10]: Given that Flux v2 can work with S3 buckets and Helm repositories only, should we reformulate "configuration in Git repositories"?

File: README.md [hunk 4]
```diff
@@ -44,16 +49,28 @@ gotk check --pre
...
+the following guides:
 
 - [Get started with GitOps Toolkit (deep dive)](https://toolkit.fluxcd.io/get-started/)
 - [Installation](https://toolkit.fluxcd.io/guides/installation/)
 - [Manage Helm Releases](https://toolkit.fluxcd.io/guides/helmreleases/)
 - [Setup Notifications](https://toolkit.fluxcd.io/guides/notifications/)
 - [Setup Webhook Receivers](https://toolkit.fluxcd.io/guides/webhook-receivers/)
 
-## Components
+## <a name="gitops-toolkit">GitOps Toolkit</a>
```
Comment (hiddeco) [inline README.md:62]: I think GitHub does this automatically? See, e.g: https://github.com/fluxcd/toolkit/#Components

Comment (squaremo) [reply README.md:62]: Oh yes, I forgot about that.

File: README.md (hunk 3 above)
Comment (squaremo) [reply README.md:10]: Yes, if there's a good replacement term -- I'm reluctant to spin sentences out further for the sake of precision. I could stretch it to "Git repositories (and similar)" but it gets a bit vague.

Comment (squaremo) [reply README.md]: Turned it around, I think it covers your point and still reads OK now.

Comment (stefanprodan) [reply README.md]: Great!

File: README.md (hunk 2 above)
Comment (squaremo) [reply README.md:1]: I think it's worth being explicit that it's a new thing, especially while fluxcd/flux is being maintained.

Comment (stefanprodan) [reply README.md:1]: Ok good point

**Comment by squaremo** [comment]
Rebass! :fish: I would think we'll wait until the repo is renamed to actually merge it.

**Review by hiddeco — APPROVED** [review]
:tada:

**Review by stefanprodan — APPROVED** [review]
Thanks @squaremo 🍰 

Please put this on hold until we rename the repo

**Comment by stefanprodan** [comment]
Ref: #388

**Pull request merged**